packages/block-editor/src/components/link-control/make-cancelable.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Creates a wrapper around a promise which allows it to be programmatically
+ * cancelled.
+ * See: https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html
+ *
+ * @param {Promise} promise the Promise to make cancelable
+ */
+const makeCancelable = ( promise ) => {
```

Comment:
Do we still need this?

Reply:
Yeh it's used in the create item flow. But we can probably revert the couple of changes that move it to a separate file?

Reply:
Done